docs/_static/css/getstarted_style.css.bak

```diff
@@ -0,0 +1,101 @@
+/* overrides */
```

Comment:
Looks like this was added under source control by mistake.

Reply:
yeah, this .bak file seems to be added by the tool automatically. Should I remove it?

Reply:
Sure, we should not store temporary files under source control.

Reply:
Thanks. Removed. :)